performdeployment.php

```diff
@@ -394,10 +422,10 @@ function ( $m ) use ( $listDictionaryURLs )
 if ( $hasSource && ! $needsLogin )
 {
 	$thisData = getThisData();
-	$hasAnyChanges = ( $thisData !== $sourceData );
+	$hasAnyChanges = false;
 	$listHasChanges = [];
 	$studyNamesMatch = true;
```

Comment:
Some people have been putting Dev or Test suffix on the end of the project names, so they will not match.  We still want to be able to deploy changes with this tool for them.

Reply:
That's why this version introduces the project names matching setting!